oda_api/cli.py

```diff
@@ -45,11 +45,18 @@ def cli(obj, debug=False, dispatcher_url=None, test_connection=False, wait=True,
 @click.pass_obj
 def get(obj, instrument, product, argument, _discover_token):
     if instrument is None:
-        logger.info("found instruments: %s", obj['dispatcher'].get_instruments_list())
+        request_token = None
+        if _discover_token:
```

Comment:
seems like this all is unnecesary if token is discovered the DispatcherAPI constructor